.mex/setup.sh

```
@@ -0,0 +1,603 @@
+esac
+
+echo ""
+
+# ─────────────────────────────────────────────────────────────
+# Step 4 — Pre-analyze codebase (if CLI available)
+# ─────────────────────────────────────────────────────────────
+
+SCANNER_BRIEF=""
+if [ "$PROJECT_STATE" != "fresh" ] && [ -n "$MEX_CMD" ]; then
+  # Run scanner in background with spinner
+  (cd "$PROJECT_DIR" && $MEX_CMD init --json 2>&1 > /tmp/mex_scanner_$$.json) &
```

Shellcheck (0.11.0)

[warning] To redirect stdout+stderr, 2>&1 must be last (or use '{ cmd > file; } 2>&1' to clarify).

(SC2069)